repository/src/main/java/org/pentaho/platform/repository2/unified/jcr/JcrRepositoryFileDao.java

```diff
@@ -844,6 +844,23 @@ public Object doInJcr( final Session session ) throws RepositoryException, IOExc
     } );
   }
 
+  public void doesFileExist( final String destRelPath ) throws Exception {
+    jcrTemplate.execute( new JcrCallback() {
+      @Override
+      public Void doInJcr( Session session ) throws IOException, RepositoryException {
```

Comment:
Can this same logic be accomplished by calling the existing `JcrRepositoryFileDao.java#internalGetFile` ?

The function `JcrRepositoryFileDao.java#internalGetFile` is ideally already tested through many releases.

Reply:
Shall we resolve this conversation please ?
we can discuss on this thread
https://github.com/pentaho/pentaho-platform/pull/5812#discussion_r1887893507

Reply:
yes we can resolve this thread, my comment is here - https://github.com/pentaho/pentaho-platform/pull/5812#discussion_r1888977812